include/dcgp/expression.hpp

```diff
@@ -283,7 +288,7 @@ class expression
      * @param[labels] The predicted outputs (a batch).
      * @param[loss_s] The loss type. Can be "MSE" for Mean Square Error (regression) or "CE" for Cross Entropy
      * (classification)
-     * @param[parallel] sets the grain for parallelism. 0 -> no parallelism n -> divides the data into n parts and evaluates them in parallel threads 
+     * @param[parallel] sets the grain for parallelism. 0 -> no parallelism n -> divides the data into n parts and evaluates them in parallel threads
```

Comment:
How are we dealing with this? I mean what happens if the user installs dcgp with no parallelism and then use the parallel option?